rpc/rpc_machine.c

```diff
@@ -468,12 +469,13 @@ static int rpc_tm_setup(struct m0_net_transfer_mc *tm,
 	if (rc < 0)
 		return M0_ERR_INFO(rc, "TM initialization");
 
+	if (m0_streq(net_dom->nd_xprt->nx_name, "lnet"))
+		max_msgs_size = m0_rpc_max_msg_size(net_dom, msg_size);
+	else
+		max_msgs_size = 1;
```

Comment:
Hm... maximal message size is 1 byte? How will this work?

Reply:
The variable name itself is misleading.
This is the: min_recv_size
which will be assigned to:
struct m0_net_transfer_mc::ntm_recv_queue_min_recv_size

The original code is using `1`.
Please indicate a reasonable value here.

Reply:
@papan-singh : Let's rename "max_msgs_size" to "min_recv_size" ?

Reply:
incorporated